auth/service/service/client.go

```diff
@@ -0,0 +1,154 @@
+package service
+
+import (
+	"context"
+
+	"github.com/tidepool-org/platform/auth"
+	"github.com/tidepool-org/platform/auth/client"
+	authStore "github.com/tidepool-org/platform/auth/store"
+	"github.com/tidepool-org/platform/errors"
+	"github.com/tidepool-org/platform/log"
+	"github.com/tidepool-org/platform/page"
+	"github.com/tidepool-org/platform/provider"
+)
+
+type Client struct {
+	*client.External
+	authStore       authStore.Store
+	providerFactory provider.Factory
+}
+
+func NewClient(cfg *client.ExternalConfig, name string, logger log.Logger, authStore authStore.Store, providerFactory provider.Factory) (*Client, error) {
+	if cfg == nil {
+		return nil, errors.New("config is missing")
+	}
+	if name == "" {
+		return nil, errors.New("name is missing")
+	}
+	if logger == nil {
+		return nil, errors.New("logger is missing")
+	}
+	if authStore == nil {
+		return nil, errors.New("task store is missing")
```

Comment:
looks like this should this be "auth store is missing" ??

Reply:
Yep, good catch. I'll fix it in the latest code rather than having to patch this and update all following PRs.